client/src/components/profile/editprofile.js

```diff
@@ -33,11 +33,22 @@ class EditProfilePopup extends Component {
 
     this.state = {
       isEditedProfileSaving: false,
-      avatar: '',
       fullname: '',
       email: '',
       gender: 'NA',
       country: 'NA',
+      avatar:'',
```

Comment:
Have these states related to avatar completely cohesive into one object called avatar or make the state completely isolated.